dockstore-webservice/src/main/resources/migrations.1.7.0.xml

```diff
@@ -44,4 +44,49 @@
     <changeSet author="louisecabansay (generated)" id="addOrganizationIdStarOrgConstratint">
         <addForeignKeyConstraint baseColumnNames="organizationid" baseTableName="starred_organizations" constraintName="fk_organizationIdStarredOrganizations" referencedColumnNames="id" referencedTableName="organization"/>
     </changeSet>
+    <changeSet author="natalieperez (generated)" id="addCommentTagTableColumnDbcreatedate">
+        <setColumnRemarks columnName="dbcreatedate" tableName="tag"/>
```

Comment:
These seem to be missing the remarks themselves, maybe forgot a commit?
Also, please group these together for readability (i.e. one changeSet with all comments being added)

Reply:
I kept them as different changesets because that's what gets generated. Otherwise, I'll fail a test on circleci.

Reply:
Oh wait, nevermind...ignore this for now.